tests/ctest/test_hooks.c

```diff
@@ -168,6 +168,33 @@ static void test_initHooks_shouldInitAllHooksWithoutFailure (void)
 	elektraFree (kdb);
 }
 
+static void test_initHooksSendNotifications(void)
```

Comment:
Ignore this test case, this was just for debugging. Tests will be added when the general design has been approved,